Assignments/Program.cs

```diff
@@ -1 +1,46 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T13 - Strong Password

```

Comment:
I think copy paste error. This is T15 assignment.
NEVER copy paste anything. ALWAYS type again.